cypress/pageobject/Shift/ShiftUpdate.ts

```diff
@@ -0,0 +1,27 @@
+export class ShiftUpdate {
+  typeShiftReason(reason: string) {
+    cy.get("#reason").click().clear().type(reason);
+  }
+  
+  typeAmbulanceDriverName(name: string) {
+    cy.get("#ambulance_driver_name").click().clear().type(name);
+  }
+  
+  typeAmbulancePhone(number: string) {
```

Comment:
- you have created duplicate variable in two different files

Reply:
But the shiftUpdate class will require the function to be slightly different than the shiftCreation one , as they will clear the previous value that was already present in  the input box . That is why i created the fuctions seperately that might seem to be duplicate . If you want i can get rid of the functions but the clearing the original values wil have to  be done in original functions , is that okay ?

Reply:
don't create duplicate functions, check out the cy.typeIntoField() implementation in the code base, where we can use same variable for inputting and clearing by passing a boolean 

Make the usage and change across your test file